docs/installation.md

```diff
@@ -1,3 +1,13 @@
+# Quick install
+
+Run below command to install latest version of aeternity node.
+
+```bash
+wget https://raw.githubusercontent.com/aeternity/epoch/master/scripts/install.sh && chmod +x install.sh && ./install.sh 1.1.0
```

Comment:
Can two CI jobs (one on Linux and one on OSX) be created (e.g. on master nightly) that:
* Check that the one line doc that user sees (either in docs/installation.md or README.md or wiki page) is the one that the CI plans to run (e.g. with `grep '^CONTENTOFTHISLINE$'`);
* Run script.
* Perform minimal successive steps for getting running node.
* Run smoke test on release.
  * For this we used to have smoke test of release package in CI - not sure if still present, may need refreshing and reenabling it.

Reply:
CircleCI is using 14.04 ..